workflow/automation/estimation/estimate_wct.py

```diff
@@ -485,15 +485,16 @@ def est_IM_chours_single(
         core_hours *= 7.5
 
     wct = core_hours / n_cores
-    if scale_ncores and wct > (
-        node_time_th_factor * n_cores / PHYSICAL_NCORES_PER_NODE
+    if scale_ncores and np.any(
+        wct > (node_time_th_factor * n_cores / PHYSICAL_NCORES_PER_NODE)
     ):
         # Make a numpy array of the input data in the right shape
         data = np.array([int(n_cores)]).reshape(1, 1)
         core_hours, wct, n_cores = scale_core_hours(
             core_hours, data, node_time_th_factor
         )
-        core_hours, n_cores = float(core_hours), int(n_cores)
+        if not hasattr(nt, "__iter__"):
+            core_hours, n_cores = float(core_hours), int(n_cores)
```

Comment:
Soft yikes, would rather have a 'single' wrapper function than start looking into dunders...

Reply:
A 'single' wrapper would also be in line with the other steps

Reply:
:cry: